fixed_frame: /world
#camera_frame: /kinect1_rgb_optical_frame
camera_offset:
  translation: [0.0, 0.0, 0.0]
  rotation:    [0.0, 0.0, 0.0, 1.0]
# There is one entry for each URDIF that should be filtered
models: 
- model: "robot_description"
  tf_prefix: ""
# how far in front of the robot model is still deleted? (e.g. 0.05 = 5cm)
depth_distance_threshold: 0.05
show_gui: false
filter_replace_value: 5.0
